miso-web/src/main/webapp/scripts/box_ajax.js

```diff
@@ -394,14 +397,18 @@ Box.ui = {
   addItemToBox: function() {
     var barcode = jQuery('#selectedBarcode').val();
     if (!jQuery('#selectedPosition').text()) {
-      jQuery('#warningMessages').html("Please select a position from the grid, then rescan the barcode.");
+      jQuery('#warningMessages').html("Please select a single position from the grid, then rescan the barcode.");
       return null;
     }
     if (Utils.validation.isNullCheck(barcode)) {
       alert("Please enter a barcode.");
     } else {
       var selectedBarcode = jQuery('#selectedBarcode').val().trim();
-      var selectedPosition = Box.utils.getPositionString(Box.visual.selected.row, Box.visual.selected.col);
+      if (Box.visual.selectedItems.length !== 1) {
+        alert('Select a single location to add the sample to');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)